docs/getting_started/building.rst

```diff
@@ -33,4 +33,4 @@ can inspect ``meson_options.txt`` which contains build-options specific to
 following :ref:`sec-building-config`.
```

Comment:
Are we missing a "gs" here - or am I misinterpreting the naming convention?

Reply:
Yes, they are inconsistently named, however, not the focus of this PR.

Reply:
You did rename a lot of them, which made me think that it was